src/reducer/videoReducer.js

```diff
@@ -0,0 +1,87 @@
+import { errorToast, successToast } from '../components/Toast/Toast'
+import {
+  ADD_TO_PLAYLIST,
+  CREATE_PLAYLIST,
+  DATA_FROM_SERVER,
+  ADD_TO_HISTORY,
+  LIKED_VIDEOS,
+  SEARCH_VIDEO,
+  WATCH_LATER
+} from '../constants/reducerConstants'
+import { checkItem } from '../utils/checkItem'
+
+export default function videoReducer(state, { type, payload }) {
```

Comment:
Nice idea to Destructure Action right in the beginning.

Reply:
:)